src/Language/utils.php

```diff
@@ -17,19 +17,21 @@ function chrUTF8(int $cp)
  */
 function ordUTF8(string $string)
 {
-    [, $ord] = unpack('N', mb_convert_encoding($string, 'UCS-4BE', 'UTF-8'));
+    [, $ord] = \unpack('N', \mb_convert_encoding($string, 'UCS-4BE', 'UTF-8'));
 
     return $ord;
 }
 
 /**
  * @param string $string
  * @param int    $position
+ * @param bool   $convertEncoding
  * @return int
  */
-function charCodeAt(string $string, int $position): int
+function charCodeAt(string $string, int $position, bool $convertEncoding = false): int
```

Comment:
Added a flag to skip converting the encoding in cases where it's not required. This is not the best solution but I think it's acceptable in this case.